src/util/timeSeries.js

```diff
@@ -1,13 +1,13 @@
-const { computeDate, getPureDate } = require('./time')
+const { computeDate, getRoundedDate } = require('./time')
 const ms = require('ms')
 
 const retentionLogDuration = '31 days'
 
 function getRetentionLimitDate () {
-  // round to pure date (0h0m0s) to have a relative stable value to display if there is any error value
+  // round to inferior UTC date with 0h0m0s to have a relative stable value to display if there is any error value
```

Comment:
Yes 'truncateDate' is a more appropriate name for this function.
The objective to truncating instead of rounding is to have at least 30 complete days of retention log.